ga4gh/cli.py

```diff
@@ -90,23 +90,48 @@ def usingWorkaroundsFor(self, workaround):
         """
         return workaround in self._workarounds
 
+
+class AbstractGetRunner(AbstractQueryRunner):
+    """
+    Abstract base class for get runner classes
+    """
+    def __init__(self, args):
+        super(AbstractGetRunner, self).__init__(args)
+        self._id = args.id
+        self._verbosity = args.verbose
+        self._httpClient = client.HTTPClient(
+            args.baseUrl, args.verbose, self._workarounds, self._key)
+
+    def _run(self, method):
+        response = method(self._id)
+        print(response.id)
+
+
+class AbstractSearchRunner(AbstractQueryRunner):
+    """
+    Abstract base class for search runner classes
+    """
+    def __init__(self, args):
+        super(AbstractSearchRunner, self).__init__(args)
+
     def setHttpClient(self, request, args):
```

Comment:
I think this should be refactored a bit:
1. Move the HTTPClient instantiation into the constructor of AbstractQueryRunner - this is shared by all the runners. Also, verbosity should be set in there.
2. Change `setHttpClient` to `_setRequest`.